core/themes/basis/js/script.js

```diff
@@ -0,0 +1,205 @@
+(function ($) {
+"use strict";
+
+Backdrop.behaviors.toggles = {
+  attach: function(context, settings) {
+    var $toggles = $('[data-toggle]');
+
+    $toggles.click(function(){
+      var $this = $(this);
+      var $target = $('[data-toggleable="' + $this.attr('data-toggle') + '"]');
+      $target.toggleClass('js-toggled');
+    });
+  }
+}
+
+// Detect if flexbox is available to give helper class for CSS
+Backdrop.featureDetect.flexbox();
+
+if (typeof Backdrop.tableDrag !== 'undefined') {
+  /**
+   * Overriding core's tableDrag prototype to update markup of handle
+   * Take an item and add event handlers to make it become draggable.
+   */
+  Backdrop.tableDrag.prototype.makeDraggable = function (item) {
```

Comment:
Oo, yikes. We can make this cleaner. Markup in JS files should run through Backdrop.theme.[whatever], so you should be able to override markup that way. But if that's missing in tabledrag.js, we should make it use `Backdrop.theme.*` and then have Basis use it. There's a lot of non-theme logic in here I'd hate to duplicate.